pages/common/ffmpeg.md

```diff
@@ -14,10 +14,18 @@
 
 `ffmpeg -i {{frame_%d.jpg}} -f image2 {{video.mpg|video.gif}}`
 
-- Extract a single frame from a video at time mm:ss and save it as a 128x128 resolution image:
+- Quickly extract a single frame from a video at time mm:ss and save it as a 128x128 resolution image:
 
-`ffmpeg -i {{video.mp4}} -ss {{mm:ss}} -frames 1 -s {{128x128}} -f image2 {{image.png}}`
+`ffmpeg -ss {{mm:ss}} -i {{video.mp4}} -frames 1 -s {{128x128}} -f image2 {{image.png}}`
```

Comment:
You seem to have just changed the order. How does this speed up the execution time of the command ?

Reply:
If the -ss is after the input file then ffmpeg re-encodes the whole file up to that point.

If -ss is specified before the input file then ffmpeg just starts from that point onwards.

See here for more details: https://trac.ffmpeg.org/wiki/Seeking

Reply:
Wow, I guess you live and learn. Thanks for the link. Also filled me with fond memories of Princess Mononoke :relaxed: